cpp/log/frontend.cc

```diff
@@ -24,6 +26,7 @@ Frontend::~Frontend() {
 }
 
 void Frontend::GetStats(Frontend::FrontendStats* stats) const {
+  lock_guard<mutex> lock(stats_mutex_);
```

Comment:
it was quicker to do this, and anyway, maybe I'll convert these to monitoring stats one day :)